Traceback (most recent call last):
  File "/home/sa7270/diffusion/run_experiment.py", line 44, in <module>
    diffusion.run()
  File "/home/sa7270/diffusion/diffusion.py", line 54, in run
    self.generate_data()
  File "/home/sa7270/diffusion/diffusion.py", line 39, in generate_data
    self.X_train.gen_rand()
  File "/home/sa7270/diffusion/data.py", line 36, in gen_rand
    self.mu=self.Î¼.numpy().to(self.device)
AttributeError: 'numpy.ndarray' object has no attribute 'to'
